examples/dapperish.go

```diff
@@ -12,7 +12,9 @@ import (
 	"runtime"
 	"strings"
 
-	"github.com/opentracing/api-golang/examples/dapperish"
+	// `go imports` was stumbling over dapperish since it's both a package name
+	// and a source file.
+	dapperish "github.com/opentracing/api-golang/examples/dapperish"
```

Comment:
oh, right, `goimports` -- I hook it up through `vim` and never invoke it manually.

TL;DR, whenever I save a file it pipes it through `goimports` and it was removing the line despite using the `dapperish` package. This fixed it. I can investigate further if you're curious.